CMakeLists.txt

```diff
@@ -754,7 +746,7 @@ if(WITH_ZLIB)
 "The chosen zlib library does not appear to be valid because it is
 missing certain required symbols. Please check that ${LIBZ_LIBRARY} is
 the correct zlib library. For details about the error, please see
-${LIBSBML_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeError.log")
+${LIBLX_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeError.log")
```

Comment:
Is a `/` needed immediately after `${LIBLX_BINARY_DIR}`?

Reply:
Since the libsbml version does not I assume this does not :-)